mesop/server/server.py

```diff
@@ -81,6 +77,28 @@ def render_loop(
         error=pb.ServerError(exception=str(e), traceback=format_traceback())
       )
 
+  def create_update_state_event(diff: bool = False) -> str:
```

Comment:
nit: I think you can pull this out as a top-level fn definition since it doesn't look like it's using closure variables.